docs/foundation/doctr.md

```diff
@@ -0,0 +1,55 @@
+[DocTR](https://github.com/mindee/doctr) is an Optical Character Recognition model.
+
+You can use DocTR with Inference to identify and recognize characters in images.
+
+### How to Use DocTR
+
+To use DocTR with Inference, you will need a Roboflow API key. If you don't already have a Roboflow account, [sign up for a free Roboflow account](https://app.roboflow.com). Then, retrieve your API key from the Roboflow dashboard. Run the following command to set your API key in your coding environment:
+
+```
```

Comment:
At this point, have we talked about how to start the roboflow inference server docker container? If now, we should mention it here and link to that part of the docs.